dashboard/app/helpers/levels_helper.rb

```diff
@@ -248,14 +248,20 @@ def app_options
     end
 
     if pairing_check_user
-      recent_driver, recent_attempt, recent_user = UserLevel.most_recent_driver(@script, @level, pairing_check_user)
-      if recent_driver && !recent_user.is_a?(DeletedUser)
-        level_view_options(@level.id, pairing_driver: recent_driver)
-        if recent_attempt
-          level_view_options(@level.id, pairing_attempt: edit_level_source_path(recent_attempt)) if recent_attempt
+      user_level = UserLevel.find_by(user: pairing_check_user, script: @script, level: @level)
+      is_navigator = !user_level.nil? && user_level.navigator?
```

Comment:
`!user_level.nil?` could be `user_level.present?`

Reply:
I didn't use `.present?` because it's not equivalent to `! .nil?` for empty string, empty object, empty array and some other cases.  I'm really surprised that `.not_nil?` was never added to ruby or rails -- I feel like this is a very basic operation that I want to do all the time!

Reply:
The way it's implemented, user_level would only be an active record or nil, right? So wouldn't it be safe to use in this case?

Reply:
Yes, I agree that it's safe for the current implementation but there's no way to guarantee the type of user_level and (to me) `! .nil?` expresses the intent better than `.present?`.  Could you elaborate a bit on why you prefer `.present?`?

Reply:
`.present?` is easier to understand than `! .nil?`

Reply:
I started a Slack thread about this: https://codedotorg.slack.com/archives/C0T0PNTM3/p1632422797381300.  Let's try Maddie's suggestion?

Reply:
Maureen and I talked through a bunch of options offline and decided that this is ok to keep as-is.